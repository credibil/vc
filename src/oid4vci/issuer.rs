//! # Credential Issuer
//!
//! API for credential issuance.

pub mod create_offer;
pub mod credential;
pub mod credential_offer;
pub mod deferred;
pub mod metadata;
pub mod nonce;
pub mod notification;
pub mod statuslist;
