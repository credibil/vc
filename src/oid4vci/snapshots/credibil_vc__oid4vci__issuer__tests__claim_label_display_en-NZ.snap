---
source: src/oid4vci/issuer.rs
assertion_line: 2602
expression: "&claims"
---
- Address.Country
- Address.Locality
- Address.Region
- Address.Street Address
- Email
- Family name
- Given name
